core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageFeatureStep.java

```diff
@@ -260,7 +260,7 @@ public void write(String s, byte[] bytes) {
         if (!proxies.isEmpty()) {
             // Needed to access DYNAMIC_PROXY_REGISTRY
             exports.produce(new JPMSExportBuildItem("org.graalvm.nativeimage.builder", "com.oracle.svm.core.jdk.proxy",
-                    GraalVM.Version.VERSION_22_1_0));
+                    GraalVM.Version.CURRENT));
```

Comment:
This should not change. It says that the export should be done for GraalVM >= 22.1